setup.py

```diff
@@ -13,7 +13,8 @@
 ]
 
 extras_require = {
-    ':python_version < "3"': 'py2-ipaddress >= 3.4.1',
+    ':python_version < "3.5"': 'backports.ssl_match_hostname >= 3.5',
+    ':python_version < "3.5"': 'ipaddress',
```

Comment:
I think this is only needed for `python < 3.3` ( https://docs.python.org/3/library/ipaddress.html )